### PR TITLE
Increase log level of a single RPC log

### PR DESCRIPTION
**Pull request by GytisCepk**
To better identify backend errors, we need a way to filter out non 5XX HTTP errors as well as see error messages in logs.

### CODE REVIEW

**Review by paulius-valiunas — APPROVED** [review]

File: core/backend/src/rpc/web/logging.ts [hunk 1]
```diff
@@ -31,15 +31,19 @@ export class WebAppRpcLoggingBackend extends WebAppRpcLogging {
...
       errorMessage,
+      error: BentleyError.getErrorProps(result),
       // Alert! The following properties are required by Bentley DevOps standards. Do not change their names!
       ActivityId: invocation.request.id, // eslint-disable-line @typescript-eslint/naming-convention
       MachineName: this.getHostname(), // eslint-disable-line @typescript-eslint/naming-convention
       ...pathIds,
-    }));
+    };
+
+    Logger.logError(CommonLoggerCategory.RpcInterfaceBackend, `${message}.${operationDescriptor}`, metadata);
```
Comment (aruniverse) [inline core/backend/src/rpc/web/logging.ts:47]: This supposed to be logError?

**Review by aruniverse — COMMENTED** [review]
Good first pass

File: core/backend/src/rpc/web/logging.ts (hunk 1 above)
Comment (GytisCepk) [reply core/backend/src/rpc/web/logging.ts:47]: Yes.

It was `logInfo`, but this is only logged on error so I'm not sure why it was only `info` (+ this is inside function called `logErrorBackend`).

**Review by pmconne — APPROVED** [review]

**Comment by aruniverse** [comment]
@mergifyio backport release/4.x

**Comment by aruniverse** [comment]
@Mergifyio backport release/4.4.x

**Pull request merged**